libtcmu.c

```diff
@@ -544,7 +544,7 @@ static void remove_device(struct tcmulib_context *ctx,
 static int is_uio(const struct dirent *dirent)
 {
 	int fd;
-	char tmp_path[64];
+	char tmp_path[NAME_MAX];
```

Comment:
Yes, you're right. I was so focused on d_name, which is of length NAME_MAX, because the gcc warning was about that. It should be PATH_MAX. I will adjust.